src/Workspaces/CoreTestUtilities/WorkspaceExtensions.cs

```diff
@@ -30,7 +30,7 @@ public static void RemoveDocument(this Workspace workspace, DocumentId documentI
             workspace.TryApplyChanges(newSolution);
         }
 
-        public static void Updatedocument(this Workspace workspace, DocumentId documentId, SourceText newText)
+        public static void UpdateDocument(this Workspace workspace, DocumentId documentId, SourceText newText)
```

Comment:
This doesn't have anything to do with the PR, just noticed that the casing was off.